### PR TITLE
Reorganized the auth ENV vars a bit.  Updated the MT openshift template

### PR DESCRIPTION
**Pull request by EricWittmann**
First step in enabling the RBAC for MAS.

### CODE REVIEW

File: ui/yarn.lock [hunk 1]
```diff
@@ -153,6 +153,13 @@
   dependencies:
     regenerator-runtime "^0.13.4"
 
+"@babel/runtime@^7.12.13":
```
Comment (carlesarnal) [inline ui/yarn.lock:156]: This is expected to be in this pr?

Comment (EricWittmann) [reply ui/yarn.lock:156]: Good point - I'll remove it.

Comment (EricWittmann) [reply ui/yarn.lock]: Done.

File: distro/openshift-template/mt/apicurio-registry-mt-template.yaml [hunk 2]
```diff
@@ -164,30 +164,37 @@ objects:
...
+          - name: REGISTRY_AUTH_ADMIN_OVERRIDE_FROM
+            value: ${REGISTRY_AUTH_ADMIN_OVERRIDE_FROM}
+          - name: REGISTRY_AUTH_ADMIN_OVERRIDE_TYPE
+            value: ${REGISTRY_AUTH_ADMIN_OVERRIDE_TYPE}
+          - name: REGISTRY_AUTH_ADMIN_OVERRIDE_ROLE
+            value: ${REGISTRY_AUTH_ADMIN_OVERRIDE_ROLE}
+          - name: REGISTRY_AUTH_ADMIN_OVERRIDE_CLAIM
+            value: ${REGISTRY_AUTH_ADMIN_OVERRIDE_CLAIM}
+          - name: REGISTRY_AUTH_ADMIN_OVERRIDE_CLAIM_VALUE
+            value: ${REGISTRY_AUTH_ADMIN_OVERRIDE_CLAIM_VALUE}
```
Comment (famartinrh) [inline distro/openshift-template/mt/apicurio-registry-mt-template.yaml:197]: I think you forgot to update the parameters section, to reflect all this changes.

Later you need to check app-interface to make sure we are using the new parameters

Comment (EricWittmann) [reply distro/openshift-template/mt/apicurio-registry-mt-template.yaml:197]: Fixed!  Thank you.

**Review by carlesarnal — APPROVED** [review]
LGTM, I agree with Fabian's points about app-interface.

**Comment by EricWittmann** [comment]
I have a MR open for the app-interface changes.  I won't link it here but I will send the link around in chat.

**Pull request merged**